HW0/P5/P5.txt

```diff
@@ -1 +1,19 @@
 # Your answers here.
+
+1. 255 seconds
+
+2. Yes. Splitting the bags evenly among the 8 cashiers, if each sums 32 bags, this will take 31 seconds. Then, 4 cashiers sum their totals with the other 4, in 1 second. 2 cashiers sum theirs with the other 2 remaining in 1 second. And the remaining 2 sum theirs in 1 second. In total, this takes 34 seconds.
+
+3. If 128 employees sum 2 bags, and then 64 employees sum their total with another cashier's total, etc.,  it will take log_{2}{256} = 8
+
+4. It will take an infinite number of employees log_2{N} seconds.
```

Comment:
I agree  this would be the ceiling of log2(n), because in case n is not a power of 2, there will be one last operation that will take a second